src/stages/evaluate.ts

```diff
@@ -216,7 +219,11 @@ export default function evaluate<T, G>(
 /**
  * Find the next enabled and pending slot in the `focusedAbility` to be prompted
  */
-function findNextSlotToPrompt<T, G>(getState: () => WolfState, abilities: Ability<T, G>[]): SlotId | null {
+function findNextSlotToPrompt<T, G>(
+  getState: () => WolfState,
+  slots: Slot<G>[],
+  abilities: Ability<T, G>[]
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1